sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/E2ETests/SqlClientTests.cs

```diff
@@ -0,0 +1,326 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// The following tests were originally copied from OpenTelemetry and modified for Azure Monitor.
```

Comment:
Had discussion with Mothra, 90% of the code in this file are from OpenTelemetry. Mothra will sort out the licensing/copyright part before we move forward on this PR.